migration/migrator/migrations/system/20230905224747_update_node_repo.py

```diff
@@ -26,4 +26,9 @@ def down(config):
     :param config: Object holding configuration details about Submitty
     :type config: migrator.config.Config
     """
-    pass
+    os.system("apt-get purge nodejs libnode72 -y")
+    os.system("rm -r /etc/apt/sources.list.d/nodesource.list")
+    os.system("rm -r /etc/apt/keyrings/nodesource.gpg")
+    os.system("umask 0022 && curl -sL https://deb.nodesource.com/setup_16.x | bash -")
```

Comment:
Should we also bump nodejs while we're at it?

Reply:
Yeah we probably should